doc/source/new-project-guide/level-4-tutorial.rst

```diff
@@ -342,6 +342,46 @@ Assume that you have successfully followed the previous steps. Now, you want to
 
 #. Done!
 
+Example usage
+^^^^^^^^^^^^^
+
+Below is a summary of the commands used at level 4 ``system`` in ``scikit-package``. See above for detailed instructions of each command.
+
+.. code-block:: bash
+
+    conda create -n skpkg_env scikit-package pre-commit
+    conda activate skpkg_env
+    cd /path/to/your/project/location
+    package create system
+        [1/6] project_name (my-project): diffraction-utils
+        [2/6] github_org (billingegroup): johndoe
+        [3/6] github_repo_name (my-project): diffraction-utils
+        [4/6] conda_pypi_package_dist_name (my-project): diffraction-utils
+        [5/6] package_dir_name (my_project): diffraction_utils
+        [6/6] project_owner_name (Simon Billinge): John Doe
+    cd diffraction-utils
+    pip install -e .
+    conda install --file requirements/test.txt
+    pytest
+    git init
+    git remote add origin https://github.com/mygithubusername/diffraction-utils.git
+    git pull origin main
+    git checkout -b skpkg-proj
+    pre-commit install
+    git add .
+    git commit -m "skpkg: start a new project with skpkg system template"
+    git push --set-upstream origin skpkg-proj
+    rm LICENSE
+    git add LICENSE
+    git commit -m "chore: remove LICENSE file created from initial GitHub repo creation"
+    git push --set-upstream origin skpkg-proj
```

Comment:
Once we  configure `--set-upstream` above, maybe this command is just `git push origin skpkg-proj`?